src/dashboard/components/dashboard-item.directive.html

```diff
@@ -19,6 +19,9 @@
   <span ng-if="::!offerCtrl.offerExpired">
     <div class="pull-right btn-group" uib-dropdown>
       <a class="btn btn-default btn-confirmed" ng-if="offerCtrl.showButtonConfirmed()" ng-click="offerCtrl.confirmEvent()" translate-once="dashboard.directive.confirm_event"></a>
+      <div class="tag-confirmed" ng-if="offerCtrl.showConfirmedTag()">
+        <span>Gaat door</span>
```

Comment:
Use `<i class="fa fa-check-circle text-success" aria-hidden="true"></i>` instead of adding it via a pseudo element. The class `text-success` will give it a green color.